packages/amplify-cli/src/commands/status.ts

```diff
@@ -1,10 +1,22 @@
+import { constructCloudWatchEventComponent } from "amplify-category-function/src/provider-utils/awscloudformation/utils/cloudformationHelpers";
+import { ViewResourceTableParams, CLIParams } from "amplify-cli-core/lib/cliViewAPI";
+
+
 export const run = async context => {
-  await context.amplify.showResourceTable();
-  await context.amplify.showHelpfulProviderLinks(context);
-  await showAmplifyConsoleHostingStatus(context);
+  const cliParams:CLIParams  = { cliCommand : context?.input?.command,
+                                 cliSubcommands: context?.input?.subCommands,
+                                 cliOptions : context?.input?.options }
+  const view = new ViewResourceTableParams( cliParams );
+  if ( context?.input?.subCommands?.includes("help")){
+     console.log( view.getStyledHelp())
+  } else {
+    await context.amplify.showStatusTable( view );
```

Comment:
Would this return gracefully when a wrong category input is passed here?

Reply:
Yes, If an incorrect category is provided, the CLI would print an empty summary table and no cloudformation-diff. We could update this to display an error message and a list of valid category types in subsequent PRs.